### ISSUE TITLE
How do I add the title of a job post to the breadcrumb menu for the detail page?

### ISSUE THREAD

**Issue by zenoussi** [itx-informationssysteme/jobapplications#127]
Example: breadcrumb navigation
`lib.breadcrumb = TYPO3\CMS\Frontend\DataProcessing\MenuProcessor
lib.breadcrumb {
    special = rootline
    special.range = 0|5
    entryLevel = 0
    as = breadcrumb
}`

Thanks a lot
Ulli

**Comment by itx-benjamin-jasper** [comment]
You should do that in the fluid template, where you have access to the posting data via `posting.title`

**Comment by zenoussi** [comment]
How can this work?
Breadcrumb-/rootline menus are created with the Menu Processor (pages) and then rendered in fluid templates of pages. Within a fluid template of a page there is no output of data records such as posting data or news data (Georg Ringer). Georg Ringer, for example, solves this using his own data processor, see https://docs.typo3.org/p/georgringer/news/11.2/en-us/Tutorials/BestPractice/BreadcrumbMenu.html

**Comment by itx-benjamin-jasper** [comment]
Yes you could define a seperate page layout, which renders without breadcrumb. The you can render it inside the detail page template and are totally free as to the presentation of job data. 

But I see the benefit of using the DataProcessor approach. Feel free to implement this and make a PR :)
